tests/TestCase/View/Helper/FlashHelperTest.php

```diff
@@ -34,8 +33,17 @@ public function setUp()
     {
         parent::setUp();
         $this->View = new View();
-        $session = new Session();
-        $this->View->request = new Request(['session' => $session]);
+
+        // load appropriate session class according to Cake version
+        if (!class_exists('\Cake\Http\Session')) {
```

Comment:
You can remove the negation and swap the `if` and `else` parts :slightly_smiling_face:

Reply:
yeah, this complexity is really useless